controllers/integrationpipeline/integrationpipeline_adapter.go

```diff
@@ -170,6 +171,9 @@ func GetIntegrationPipelineRunStatus(adapterClient client.Client, ctx context.Co
 	}
 
 	if !outcome.HasPipelineRunPassedTesting() {
+		if !outcome.HasPipelineRunValidTestOutputs() {
+			return intgteststat.IntegrationTestStatusTestFail, strings.Join(outcome.GetValidationErrorsList(), "; "), nil
```

Comment:
We could also add a new status if that would make things easier, but so far I was able to avoid it.

Reply:
Agreed, let's keep it as failure for now and we can see how the users feel about it before introducing a new status.